cuvs-bench/gpu/Dockerfile

```
@@ -29,7 +29,7 @@ RUN apt-get install perl -y
 RUN <<EOF
 mamba update --all -y -n base
 mamba install -y -n base \
-    "raft-ann-bench=${RAPIDS_VER}.*" \
+    "cuvs_bench=${RAPIDS_VER}.*" \
```

I wish we had named this package `cuvs-bench` instead of `cuvs_bench`... that would be more favorable to align with how most packages are named in RAPIDS (and the conda ecosystem generally).

---

I wish so too :( 

@divyegala @cjnolet  I wonder if this would be something we should try to change, since we have had only one release with the `_` named packagE?

---

I 100% agree with this- cuvs-bench seems more natural to me (and matches  the code examles in our docs.. )

---

Yes that's totally fine. Keep in mind that the actual Python package still must be called `cuvs_bench`, even if we call the conda/pip packages `cuvs-bench`. Here's the historical reason why: https://github.com/rapidsai/raft/pull/2333

TL;DR A python module called `cuvs-bench` is not a valid Python identifier

---

Yes, that's correct. We use dashes for package names and underscores for import names. e.g. `conda install dask-cudf`; `import dask_cudf`.

---

I attempted to fix this here: https://github.com/rapidsai/cuvs/pull/417

---

Update from https://github.com/rapidsai/cuvs/pull/417.
```suggestion
    "cuvs-bench=${RAPIDS_VER}.*" \
```